packages/presets/src/ai/messages/_common/generating-placeholder.ts

```diff
@@ -31,12 +35,16 @@ class AIGeneratingPlaceholder extends WithDisposable(LitElement) {
   height: number = 300;
 
   protected override render() {
+    const theme = getThemeMode();
```

Comment:
We may need to listen to `mediaQuery` to get the latest theme in real time.

Reply:
This is ok, because we can open settings when the ai-pannel opened

Reply:
User may open the system settings.